en/rest/schemas.mdown

```diff
@@ -0,0 +1,186 @@
+# Schema
+
+This API allows you to access the Schema of your app.
+
+
+## Fetch Schema
+To fetch the Schema for all the classes of your app run:
+```bash
+curl -X GET \
+  -H "X-Parse-Application-Id: ${APPLICATION_ID}" \
+  -H "X-Parse-REST-API-Key: ${REST_API_KEY}" \
+  -H "Content-Type: application/json" \
+  https://api.parse.com/1/schemas
+```
+```python
+import json,httplib
+connection = httplib.HTTPSConnection('api.parse.com', 443)
+connection.connect()
+connection.request('GET', '/1/schemas', '', {
+       "X-Parse-Application-Id": "${APPLICATION_ID}",
+       "X-Parse-REST-API-Key": "${REST_API_KEY}",
+       "Content-Type": "application/json"
+     })
+result = json.loads(connection.getresponse().read())
+print result
+```
+
+The response body is JSON containing all the schema information of the app.
+
+```json
+{
+  "results": [
+    {
+      "className": "Game",
+      "fields": {
+        "ACL": {
+          "type": "ACL"
+        },
+        "createdAt": {
+          "type": "Date"
+        },
+        "objectId": {
+          "type": "String"
+        },
+        "name": {
+          "type": "String"
+        },
+        "score": {
+          "type": "Number"
+        },
+        "updatedAt": {
+          "type": "Date"
+        }
+      }
+    },
+    ...
+  ]
+}
+```
+
+To fetch schema of a particular class run
+
+```bash
+curl -X GET \
+  -H "X-Parse-Application-Id: ${APPLICATION_ID}" \
+  -H "X-Parse-REST-API-Key: ${REST_API_KEY}" \
+  -H "Content-Type: application/json" \
+  https://api.parse.com/1/schemas/Game
+```
+```python
+import json,httplib
+connection = httplib.HTTPSConnection('api.parse.com', 443)
+connection.connect()
+connection.request('GET', '/1/schemas/Game', "", {
+       "X-Parse-Application-Id": "${APPLICATION_ID}",
+       "X-Parse-REST-API-Key": "${REST_API_KEY}",
+       "Content-Type": "application/json"
+     })
+result = json.loads(connection.getresponse().read())
+print result
+```
+
+## Adding a class
+
+To add a new class to your schema run.
```

Comment:
Needs : 

This sentence is confusing because earlier you said fetch a schema, now you say add a class to your schema. So schema and class are referring to the same thing. I think here you need to say 'add a new schema'. But this sentence could use some more explanation of what this actually does.